README.md

```diff
@@ -111,6 +111,36 @@ kubectl get pods -n codewind
 If successful, you should see the codewind-operator pod running and ready for work.
 
 
+## Persistent storage requirements
+
+Keycloak and Codewind pods have storage requirements. Both require available PersistentStorage to be configured and available before you attempt to deploy each service.
+
+Each Keycloak instance requires by default:
+
+- 1Gi capacity
+- access mode of RWO (ReadWriteOnly)
+
+Each Codewind instance requires by default:
+
+- 10Gi capacity
+- access mode of RWX (ReadWriteMany)
+
+Before continuing, ensure your cluster has the necessary `Persistent Volume` entries available for claiming. If your cluster is not using dynamically assigned storage, you can check the available status by using the command : `kubectl get pv`
+
+In this example there are three Persistent Volumes available one sized 1Gi (mode RWO) and two sized 10Gi (mode RWX) which will allow for one new Keycloak and two new Codewind deployments.
+
```

Comment:
Commas required, one after '...three persistent volumes available, one ...', and the other after 'two sized 10Gi (mode RWX), so sentence reads:

In this example there are three Persistent Volumes available, one sized 1Gi (mode RWO) and two sized 10Gi (mode RWX), which will allow for one new Keycloak and two new Codewind deployments.